docs/devguide/pull-request-guidelines.asciidoc

```diff
@@ -0,0 +1,129 @@
+[[pr-review]]
```

Comment:
Let's move this to a separate PR for discussion and focus this PR only on the template.

Reply:
Done